index.js

```diff
@@ -32,19 +29,11 @@ function compareLastModifiedTime(stream, sourceFile, targetPath) {
 		});
 }
 
-// Only push through files with different SHA1 than the destination files
+// Only push through files with different content than the destination files
 function compareSha1Digest(stream, sourceFile, targetPath) {
```

Comment:
Can you rename this to `compareContent` and update the docs. And create an alias `compareSha1Digest` => `compareContent`.